src/utils/tests/errors.rs

```diff
@@ -21,7 +21,7 @@ async fn api_error_responder_json() {
     let mut response = api_error.respond_to(&request).unwrap();
 
     let status_code: u16 = response.status().code;
-    let body_json = &response.body_string().await.unwrap();
+    let body_json = &response.body_mut().to_string().await.unwrap();
```

Comment:
oh it kind of sucks that they got rid of that method I suppose.

Reply:
I actually like the new one more, as it is more explicit what is happening. Before it was really clear that you are consuming the response.